frontend/src/components/mine/Applications/MineApplicationTable.js

```diff
@@ -0,0 +1,137 @@
+ */
+
+const propTypes = {
+  applications: PropTypes.arrayOf(CustomPropTypes.application),
+  applicationStatusOptions: PropTypes.arrayOf(CustomPropTypes.dropdownListItem).isRequired,
+  major_mine_ind: PropTypes.bool.isRequired,
+  openEditApplicationModal: PropTypes.func.isRequired,
+};
+
+const defaultProps = {
+  applications: [],
+};
+
+const columns = [
+  {
+    title: "Application No.",
+    dataIndex: "applicationNo",
+    key: "applicationNo",
+    render: (text) => <div title="Application No.">{text}</div>,
+  },
+  {
+    title: "Status",
+    dataIndex: "status",
+    key: "status",
+    render: (text, record) => (
+      <div title="Status">
+        {record.applicationStatusOptions.find((item) => item.value === text).label}
+      </div>
+    ),
+  },
+  {
+    title: "Received Date",
+    dataIndex: "receivedDate",
+    key: "receivedDate",
+    render: (text) => <div title="Received Date">{text}</div>,
+  },
+
+  {
+    title: "Description",
+    dataIndex: "description",
+    key: "description",
+    render: (text) => (
+      <div title="Description">
+        <p className="wrapped-text" style={{ maxWidth: "800px" }}>
+          {text}
+        </p>
+      </div>
+    ),
+  },
+  {
+    title: "",
+    dataIndex: "applicationEdit",
+    key: "applicationEdit",
+    align: "right",
+    render: (text, record) => (
+      <AuthorizationWrapper inTesting>
+        <AuthorizationWrapper permission={Permission.CREATE} isMajorMine={text.major_mine_ind}>
+          <Button
+            className="permit-table-button"
+            type="ghost"
+            onClick={(event) => record.openEditApplicationModal(event, text.application)}
+          >
+            <div>
+              <img className="padding-small--right icon-svg-filter" src={EDIT_OUTLINE} alt="Edit" />
+            </div>
+          </Button>
+        </AuthorizationWrapper>
+      </AuthorizationWrapper>
+    ),
+  },
+];
+const transformRowData = (
+  application,
+  major_mine_ind,
+  applicationStatusOptions,
+  openEditApplicationModal
+) => {
+  return {
+    key: application.permit_guid,
+    applicationNo: application.application_no || Strings.EMPTY_FIELD,
+    status: application.application_status_code || Strings.EMPTY_FIELD,
+    receivedDate: application.received_date || Strings.EMPTY_FIELD,
+    description: application.description || Strings.EMPTY_FIELD,
+    applicationEdit: {
+      application,
+      major_mine_ind,
+    },
+    applicationStatusOptions,
+    openEditApplicationModal,
+  };
+};
+
+export const MineApplicationTable = (props) => {
+  const rowData = props.applications.map((application) =>
+    transformRowData(
+      application,
+      props.major_mine_ind,
+      props.applicationStatusOptions,
+      props.openEditApplicationModal
+    )
+  );
+
+  return (
+    <Table
+      className="nested-table"
+      rowClassName={() => "table-row-align-middle"}
+      align="left"
+      pagination={false}
```

Comment:
className `table-role-align-middle` and the next line is `align="left"`

Are these targeting the same alignment?

Reply:
Isn't one of those applying to the table and one is applying to the row in the table?

Reply:
I'm not sure on that, never used the `rowClassName` before.
Is this table supposed to look different than the others? If it's supposed to be the same, Id removes the `rowClassName`, and use the `align="left"`

If not, disregard this comment.

Reply:
I can't speak to how it's being used in this context, but I introduced `rowClassName` to the Permit Table when dealing with nested tables. Because each 'row' was a table, it needed custom classes to deal with on-hover behaviour.

If there's nothing special about this table then `rowClassName` is _likely_ not needed

Reply:
I brought this over from the permit table I was just trying to keep them the same, Im not super sure what this is doing. I feel like its applying the class to the row content but I could be wrong.

Reply:
ahhhhh, that makes sense. I'll remove it.

Reply:
Thanks!

Reply:
It's likely not needed here. I used this as it was a clean way to align buttons consisting of mixed parts (Icon, button, svg image) that were also taller than the default row height causing weird spacing on the permit table. The alternative looked like a bunch of padding/margin overrides. We'll require conditional classes on the permit table to highlight amalgamated permits, so I thought using the rowClassName hook made sense because of that too.